src/main/java/org/openmainframeproject/cobolcheck/workers/Generator.java

```diff
@@ -126,6 +126,8 @@ private void processingBeforeEchoingSourceLineToOutput() throws IOException {
             workingStorageHasEnded = true;
         }
         if (interpreter.didLineJustEnter(Constants.PROCEDURE_DIVISION) && interpreter.currentLineContains(Constants.PROCEDURE_DIVISION)){
```

Comment:
Having file section just before procedure division, is something I have never tried. 
Does this work on Z? 
I wont be able to test before the 14th.